tool/build_runner_build.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash --
```

Comment:
This leaves Windows people out.
Why not write it as a dart script?

Reply:
Done.

It's still not going to work on Windows because it needs the pub cache location, but it's a lot closer to working on Windows :)

PTAL.